README.md

```diff
@@ -147,14 +172,8 @@ We are also in the process of plugging in Structured Logging which is the depart
 
 ### 3. Session handling
 
-We use the department-wide [Session Handler](https://github.com/companieshouse/node-session-handler) for managing session data. In order to make it's integration into the app more transparent and linear, there's a wrapper around it and can be found at `server/lib/Session.js`
-
-Presently, this service is not yet behind sign in but eventually will be . When this happens, the session wrapper implementation might need a few tweaks to account for this change.
+We use the department-wide [Session Handler](https://github.com/companieshouse/node-session-handler) for managing session data.
 
-### To-do
+### 4. Application List configuration
 
-- [] Integrate Structured Logging
-- [] Form-based validation
-- [] Multple form submissions
-- [] Lock down major dependencies to specific versions that are proven to work well
-- [] Repurpose for Typescript
+There is a maximum batch size for the number of applications (currently set to 1,000,000) which is stored a variable, called DEFAULT_BATCH_SIZE. It can be found, and it's value edited, in `server/config/.env`.

```

Comment:
its really minor but would it be better to mention the 1 million number is per user?

Reply:
Its per query not per user.
Also it applies to Keys as well as clients

Reply:
Changed